example_test.go

```diff
@@ -24,6 +24,24 @@ func example_connect() *tarantool.Connection {
 	return conn
 }
 
+// Example demonstrates how to use SSL transport.
+func ExampleSslOpts() {
```

Comment:
Shouldn't there be any restrictions? What happens when we pass SSL options while connecting to a Tarantool without SSL support?

Reply:
I think no, it's just an example for godoc. It does not run with tests ( becase no "// Output:" comment).

The example will panic with an error if you run it mannually and SSL is not supported by Tarantool. But it's ok. It will panic with an invalid port too.